micrometer/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector
+    command: [--config=/etc/otel-collector-config.yaml]
+    volumes:
+      - ./otel-collector-config.yaml:/etc/otel-collector-config.yaml
```

Comment:
This volume mount doesn't work to me. I get a error:
```
collector server run finished with error: failed to resolve config: cannot resolve the configuration: cannot retrieve the configuration: unable to read the file file:/etc/otel-collector-config.yaml: open /etc/otel-collector-config.yaml: permission denied
```
I try to add `:Z` at end of this line and it helped to me resolve the permission error.
```
- ./otel-collector-config.yaml:/etc/otel-collector-config.yaml:Z
```